war/src/main/webapp/css/style.css

```diff
@@ -1858,19 +1882,20 @@ table#legend-table td {
 }
 
 /* Color overrides */
+#jenkins .yui-button.primary .btn,
 #jenkins .yui-button.primary button {
     background-color: #4b758b;
     color: #eee;
     border: 1px solid #5788a1;
 }
-
+#jenkins .yui-button.primary .btn:hover,
 #jenkins .yui-button.primary button:hover,
 #jenkins .yui-button.primary .hover
 {
     background-color: #3f6275;
     border: 1px solid #5788a1;
 }
-
+#jenkins .yui-button.primary .btn:active,
 #jenkins .yui-button.primary button:active,
 #jenkins .yui-button.primary .active
 {
```

Comment:
The formatting changes (why do people continue to do this?) make the `style.css` changes harder to read, but I have to wonder why they need to go in here at all. Are these style changes specific to Add Item? If they are, I think they need to go into the add-item specific CSS.